pkg/config/template.go

```diff
@@ -26,6 +26,11 @@ import (
 
 	"k8s.io/client-go/kubernetes/scheme"
 	corev1 "k8s.io/client-go/kubernetes/typed/core/v1"
+
+	prowgithub "k8s.io/test-infra/prow/github"
+	_ "k8s.io/test-infra/prow/hook"
```

Comment:
I should add a non-validating load so we don't need this -.-

Reply:
Does the configuration check we have also validate plugins? If not (and even if it is?), validating this may be an additional benefit of `rehearse`.

Reply:
It does, yeah